scripts/mkits-tlt-rutx-fit.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+#
+# Licensed under the terms of the GNU GPL License version 2 or later.
+# Author: Samuele Barella <dotsynergy@proton.me>, based on mkits-qsdk-ipq-image.sh and mkits.sh from Teltonika's RUTX SDK.
```

Comment:
COuld you please extend the existing `mkits.sh` script to support multiple dto files instead.